middleware/domain/source/manager/admin/cli.cpp

```diff
@@ -781,11 +788,13 @@ note: not all options has legend, use 'auto complete' to find out which legends
                         { "domain.manager.server.instances.running", string::compose( instances_count( state.servers, running))},
                         { "domain.manager.server.instances.scale.out", string::compose( instances_count( state.servers, scale_out))},
                         { "domain.manager.server.instances.scale.in", string::compose( instances_count( state.servers, scale_in))},
+                        { "domain.manager.server.instances.missing", string::compose( instances_difference( state.servers, all, running))},
```

Comment:
Is `missing` the best name for this? Feels like it doesn't completely describes what it is. Hmm, maybe it's ok. 
Since we'll be living with this a long while :) It's probably worth a few brain cycles to discuss this more.

Reply:
Apart from my reservation about the naming of the information stuff, it's looks good.

Reply:
I took it from the term we use for forward instances, though maybe we don't expose that term to the user. Either way I'm open to better terminology

Reply:
`exited` ? At least it's more telling than `missing` i think. 
Do you only count exited != `0` as _missing_? If so, we could use `failed`?

Reply:
I'm not sure... Let's go with `exited`? It's ok with `missing` if you prefer it.

Reply:
Hmm, I think I'm missing the point :) exited is not correct :). `missing` is actually much better.  Sorry for wasting your time :)